test/color_laws.jl

```diff
@@ -248,3 +249,40 @@ end
         @test ustrip.(reddening) ≈ ref_values[rv] rtol = 0.016
     end
 end
+
+@testset "F99" begin
+    # From Fitzpatrick (1999) Table 3
+
+    x_inv_microns = [0.377, 0.820, 1.667, 1.828, 2.141, 2.433, 3.704, 3.846]
+    wave = 1e4 ./ x_inv_microns
+
+    ref_values = Dict(
+        3.1 => [0.265, 0.829, 2.688, 3.055, 3.806, 4.315, 6.265, 6.591] ./ 3.1,
+    )
+
+    # test defaults
+    @test F99().(wave) ≈ ref_values[3.1] rtol = 0.016
+
+    for rv in keys(ref_values)
+        law = F99(Rv = rv)
+        output = @inferred map(law, wave)
+        @test output ≈ ref_values[rv] rtol = 0.016
+
+        bad_xs = [0.2, 11.0]
+        @test @inferred(map(law, bad_xs)) == zeros(length(bad_xs))
+        @test_throws ErrorException f99_invum(bad_xs[1], rv)
+        @test_throws ErrorException f99_invum(bad_xs[2], rv)
+
+        # uncertainties
+        #noise = randn(length(wave)) .* 0.01
+        #wave_unc = wave .± noise
+        #reddening = @inferred map(law, wave_unc)
+        #@test Measurements.value.(reddening) ≈ ref_values[rv] rtol = 0.016
```

Comment:
It's weird to use `randn` for the noise, as you can get negative errors, which is not great (`Measurements.jl` doesn't enforce non-negative errors), unless you take the absolute value.  With this change you should be able to test propagation of the uncertainty through `Dierckx.jl`
```suggestion
        noise = rand(length(wave)) .* 0.01
        wave_unc = wave .± noise
        reddening = map(w -> @uncertain(law(w)), wave_unc)
        @test Measurements.value.(reddening) ≈ ref_values[rv] rtol = 1e-3
```

Reply:
Awesome, thanks! Should this change also be made for the other models' tests?

Reply:
If by "this change" you refer to `randn` -> `rand` for the noise, I'd say probably yes.

Reply:
Yep, that was all! Ok, would switching everything over to `rand` be better as a separate PR?

Reply:
> would switching everything over to rand be better as a separate PR?

Yes, I think it's better to keep logically separate changes in different pull requests :wink: